pkg/cmd/taskrun/list_test.go

```diff
@@ -206,6 +229,12 @@ func TestListTaskRuns(t *testing.T) {
 			args:      []string{"list", "--reverse", "-n", "foo", "-o", "jsonpath={range .items[*]}{.metadata.name}{\"\\n\"}{end}"},
 			wantError: false,
 		},
+		{
+			name:      "print taskrunsruns in all namespaces",
+			command:   command(t, trsMultipleNs, now, ns),
+			args:      []string{"list", "--all-namespaces", "-n", "foo"},
```

Comment:
My thought is here that you didn't mean to put in `-n foo` since this is supposed to be a test for `--all-namespaces`. 

But this is something that should be tested for. There should be an error in a situation like this since `--all-namespaces` and `-n` aren't compatible. It would be good to have an additional test for this scenario.

Reply:
I'm now seeing your comment from above. The validation I'm mentioning here should be done before the namespace check.

Reply:
I will update with this check.